luigi/process.py

```diff
@@ -97,11 +99,16 @@ def daemonize(cmd, pidfile=None, logdir=None, api_port=8082, address=None, unix_
     stdout_proxy = open(stdout_path, 'a+')
     stderr_proxy = open(stderr_path, 'a+')
 
-    ctx = daemon.DaemonContext(
-        stdout=stdout_proxy,
-        stderr=stderr_proxy,
-        working_directory='.'
-    )
+    daemon_kwargs = {
+        "stdout": stdout_proxy,
+        "stderr": stderr_proxy,
+        "working_directory": '.',
+    }
+
+    if (StrictVersion(gdvi().get("version") or "0.0.0") >= StrictVersion("2.1.0")):
```

Comment:
I thought that using StrictVersion from distutils to compare versions was the way to go.

>>> from distutils.version import StrictVersion
>>> StrictVersion("1.10") > StrictVersion("1.2")
True